doc/nrf/app_power_opt.rst

```diff
@@ -0,0 +1,303 @@
+It is because most of the samples and applications in |NCS| are configured to perform logging over serial port by default.
+
+
+Below is the power measurement output on Power Profiler Kit II for an nRF9160 DK with the :ref:`zephyr:blinky-sample` sample compiled for the ``nrf9160dk_nrf9160`` build target without modifications in the sample configuration.
+
+.. figure:: images/app_power_opt_blinky_serial_on.png
+      :scale: 50 %
+      :alt: Current measurement on the Blinky sample with serial logging enabled
+
+      Current measurement for the Blinky sample with serial logging enabled
+
+In the image, you can see that the average current is close to 600 µA, which drains a 500 mAh lithium polymer battery in a month approximately.
+Hence it is recommended to disable serial logging to reduce current consumption.
+
+To disable serial output, you must change the project configuration associated with the sample or application.
+|config|
+
+1. Set the project configuration ``CONFIG_SERIAL`` to ``n`` irrespective of whether you are building the sample for the secure or non-secure build targets.
+2. For the non-secure build target (``nrf9160dk_nrf9160ns``), you must ensure that serial logging is disabled in :ref:`secure_partition_manager` also. To disable serial logging in Secure Partition Manager, complete the following steps:
+
+   a. Add a :file:`spm.conf` file in the project directory with the following content:
+
+      .. code-block:: console
+
+	      CONFIG_IS_SPM=y
+	      CONFIG_FW_INFO=y
+	      CONFIG_GPIO=y
+	      CONFIG_SERIAL=n
+
+
+   #. Add the entry ``set(spm_CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/spm.conf)`` in the :file:`CMakeLists.txt` file.
+
+
+The following output on Power Profiler Kit II shows the power consumption on an nRF9160 DK with the sample compiled for the ``nrf9160dk_nrf9160`` build target with ``CONFIG_SERIAL=n``.
+
+.. figure:: images/app_power_opt_blink_serial_off.png
+      :scale: 50 %
+      :alt: Current measurement on the Blinky sample with serial logging disabled
+
+      Current measurement on the Blinky sample with serial logging disabled
+
+Observe that the average current reduces to 4 µA, which implies 14 years of battery life on a 500 mAh lithium polymer battery compared to the 30-day battery life of the previous measurement.
+
+For a similar configuration, see the :ref:`udp` sample, which is optimized for low power operation on the ``nRF9160dk_nrf9160ns`` build target without any modifications.
+
+
+Idle current due to other peripherals
+=====================================
+
+Elevated currents can be caused by peripherals other than the serial ports.
+
+The power management of the Nordic SoCs automatically switches in and out the resources that are needed by the active peripherals.
+Peripherals that need a high frequency clock like UART, PWM and PDM, will show similar elevated currents if the high frequency timers associated with these peripherals are not turned off during sleep.
```

Comment:
`high frequency clock` and `high frequency timer` should be aligned.
I was a little confused by this and the previous sentence. The first states that the power management automatically enables and disables resource required by the peripherals, while the latter says that the users must control the clocks required by peripherals, or am I reading the last sentence wrong?

Reply:
What about replacing the last sentence with: "Peripherals that need a high frequency clock like i.e UART, PWM, PDM or high frequency timers, will give similar currents when enabled"